### PR TITLE
Using the higher entropy uniqid

### PR DESCRIPTION
**Pull request by mikeSimonson**

### CODE REVIEW

File: tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ class InvalidArgumentExceptionTest extends PHPUnit_Framework_TestCase
 {
     public function testFromNonExistingTypeWithNonExistingClass()
     {
-        $className = __CLASS__ . uniqid();
+        $className = __CLASS__ . uniqid('', true);
```
Comment (Ocramius) [inline tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php]: Can't really work due to `.` being added to the string

Comment (mikeSimonson) [inline tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php]: For my defence the test pass ...

Comment (Ocramius) [inline tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php]: Yes, but the class name is invalid. Can you add a `str_replace` around it, please?

Comment (mikeSimonson) [inline tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php]: I though I did, did I push ?

Comment (mikeSimonson) [inline tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php]: should I do that in the getInvalidClassNames too ?

Comment (Ocramius) [inline tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php]: Ah, yes, seeing it, although you overwrote the variable: naughty!

**Comment by Ocramius** [comment]
@mikeSimonson we can most probably drop 5.3.3 from the matrix now

**Comment by Ocramius** [comment]
We should probably drop all EOL'd versions, tbh - means keeping only 7.0, 7.1 and 7.2 (`nightly`)

**Comment by mikeSimonson** [comment]
I have done the travis update in another pr.

**Pull request merged**